GpsApp/Top/Topology.cpp

```diff
@@ -128,66 +131,105 @@ void constructApp(int port_number, char* hostname, char* device) {
     rateGroup2Comp.init(10,1);
     
     rateGroup3Comp.init(10,2);
+
+    // Initialize block driver
+    blockDrv.init(10);
+
     //GPS-- Here we initialize the component with a queue size, and instance number. The queue size governs how
     //      many waiting port calls can queue up before the system asserts, and the instance number is a unique
     //      number given to every instance of a given type.
     gpsSerial.init(1);
     gpsImpl.init(10, 1);
+
 #if FW_ENABLE_TEXT_LOGGING
     textLogger.init();
 #endif
 
     eventLogger.init(10,0);
-
+    
     linuxTime.init(0);
 
     chanTlm.init(10,0);
 
     cmdDisp.init(20,0);
 
     cmdSeq.init(10,0);
-    cmdSeq.allocateBuffer(0,seqMallocator,5*1024);
+    cmdSeq.allocateBuffer(0,mallocator,5*1024);
 
     prmDb.init(10,0);
 
-    groundIf.init(0);
-    socketIpDriver.init(0);
-
+    comm.init(0);
+    downlink.init(0);
+    uplink.init(0);
     fileUplink.init(30, 0);
     fileDownlink.init(30, 0);
+    fileDownlink.configure(1000, 1000, 1000, 10);
+    fileManager.init(30, 0);
     fileUplinkBufferManager.init(0);
-    fileDownlinkBufferManager.init(1);
     fatalAdapter.init(0);
     fatalHandler.init(0);
     health.init(25,0);
+    //pingRcvr.init(10);
```

Comment:
Could you possibly remove the other commented out references to `pingRcvr` and other commented out code in the topology file?